src/_playground/App.scss

```diff
@@ -95,6 +95,7 @@ $menu-spacing: 20px;
     background-color: #EDEFF0;
     display: flex;
     flex-direction: column;
+    overflow-y: auto;
```

Comment:
By getting rid of previous custom CSS for frDocs-Nav, overflow was no longer an attribute for side-nav.  Larger question for Fiori Fundamentals - do we want to add an overflow option to side-nav to scroll through list? Right now, the list is not scrollable unless you put custom CSS in.

Reply:
I think this got added to the wrong container class.  See the before and after screen captures below.  The scrolling region should only be the side navigation component and should not extend into the search field or the top header.

#### Before (Correct)
![fd-react-side-nav-scrolling-before](https://user-images.githubusercontent.com/11407353/55744127-c8acb000-59f9-11e9-8099-129a8fbd334a.gif)

#### After (Incorrect)
![fd-react-side-nav-scrolling-after](https://user-images.githubusercontent.com/11407353/55744137-cf3b2780-59f9-11e9-8200-53134be79785.gif)